compose/ui/ui-test/src/skikoMain/kotlin/androidx/compose/ui/test/InputDispatcher.skikoMain.kt

```diff
@@ -47,13 +52,14 @@ internal class SkikoInputDispatcher(
 ) {
 
     private val scene get() = root.scene
-    private var batchedEvents = mutableListOf<() -> Unit>()
+    private var currentClockTime = currentTime
```

Comment:
I'm not sure why this is needed, instead of just using `testContext.testOwner.mainClock.currentTime`, but this is how it's done in Android, so I decided it's safer to replicate it.